src/Appwrite/Platform/Services/Tasks.php

```diff
@@ -29,6 +30,7 @@ public function __construct()
             ->addAction(Doctor::getName(), new Doctor())
             ->addAction(Install::getName(), new Install())
             ->addAction(Maintenance::getName(), new Maintenance())
+            ->addAction(CreateMissingSchedules::getName(), new CreateMissingSchedules())
```

Comment:
Let add a patch prefix for this command so we know how to treat it and remove it when times comes.

Reply:
Renamed to `PatchCreateMissingSchedules`.